docs/server-config.md

```diff
@@ -4,47 +4,149 @@
 
 <!-- toc -->
 
+- [Custom Settings](#custom-settings)
+  * [Configuration File Content](#configuration-file-content)
+  * [Configuration File Example](#configuration-file-example)
 - [Configuring HTTP vs HTTPS](#configuring-http-vs-https)
 - [Authentication](#authentication)
   * [Enabling Authentication](#enabling-authentication)
   * [Password File](#password-file)
-  * [Setting Up Environment](#setting-up-environment)
   * [Adding an Account](#adding-an-account)
   * [Updating an Account](#updating-an-account)
   * [Deleting an Account](#deleting-an-account)
 - [Logging](#logging)
   * [Request Context Logging](#request-context-logging)
-- [Custom Script Timeout](#custom-script-timeout)
 
 <!-- tocstop -->
 
 <!-- markdownlint-enable MD004 -->
 
-Default settings for TabPy may be viewed in the
-tabpy_server/common/default.conf file. This file also contains
-commented examples of how to set up your TabPy server to only
-serve HTTPS traffic and enable authentication.
+## Custom Settings
 
-Change settings by:
+TabPy starts with set of default settings unless settings are provided via
```

Comment:
TabPy starts with a set of default settings unless settings are provided via environment variables or with a config file.

Reply:
fixed